metis-indexing/src/main/java/eu/europeana/indexing/SettingsConnectionProvider.java

```diff
@@ -70,7 +72,8 @@ public SettingsConnectionProvider(IndexingSettings settings)
     // Create mongo connection.
     try {
       this.mongoClient = createMongoClient(settings);
-      this.mongoServer = setUpMongoConnection(settings, this.mongoClient);
+      this.edmMongoClient = setUpEdmMongoConnection(settings, this.mongoClient);
+      this.recordRedirectDao = setUpRecordRedirectDaoConnection(settings, this.mongoClient);
```

Comment:
This assumes that the redirect and edm mongo databases are always on the same server. Is this a valid assumption??

Reply:
Yes this is a valid assumption. At least so far.